source/TilegramData.js

```diff
@@ -0,0 +1,22 @@
+import pitchPopulationTilegram from '../tilegrams/pitch-us-population-500k.json'
+
+class TilegramData {
+  constructor() {
+    this._tilegrams = [
+      {
+        label: 'Pitch U.S. Population 2016',
```

Comment:
Oh, good point. Does the year imply the beginning or end of that year? Because the map was made this year—so it may still make sense to keep that as the label—but we should triple-check the data.